practice_app/pages/api/getTranslation.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios')
+const { checkTranslationResponse } = require('./getTranslationModule')
+
+export default async (req, res) => {
+    // req object contains whatever information we can get from the request that comes from the client (from the browser)
+    // res object is used tow write our reponse
+    const { query, method } = req
+
+    switch (method) {
+        case 'GET':
+            {
+                console.log(query)
+                let target = query.countryCode == null ? '' : query.countryCode
+                target = target.toLowerCase()
+                const text = query.text == null ? '' : query.text
+                let sourceLang = query.sl == null ? '' : query.sl
```

Comment:
I have set source language to spanish, target language to turkish and tried to translate 'process'. And api converted it from english to turkish. Looks like it automatically decides how source language if it is not found in given. Maybe specifying source language can be unnecessary.

Reply:
I could not reproduce this. When i try, it translates from Spanish and the meaning is same as in English, "surec".